src/main/resources/messages.properties

```diff
@@ -1047,10 +1031,6 @@ doc-upload-signpost.well-ask-about=We'll tell you
 doc-upload-signpost.how-to-add-docs=How to add docs
 doc-upload-signpost.which-docs=Which docs you can add
 doc-upload-signpost.how-to-submit=How to submit now or later
-# doc upload recommendations
```

Comment:
Assuming these are being removed intentionally but noting that it doesn't seem like these message properties were part of any screens that were removed?